NICE.Identity.Management/ClientApp/src/views/UsersList/UsersList.tsx

```diff
@@ -1,42 +1,103 @@
-import React, { FunctionComponent } from "react";
+import React, { Component } from "react";
+
 import { Link } from "react-router-dom";
-import { useFetch } from "../../hooks/hooks";
+import { Breadcrumbs, Breadcrumb } from "@nice-digital/nds-breadcrumbs";
+import { Card } from "@nice-digital/nds-card";
+import { Tag } from "@nice-digital/nds-tag";
+
+import { Endpoints } from "../../data/endpoints";
+
 import { Filter } from "../../components/Filter/Filter";
 
-export const UsersList: FunctionComponent = () => {
-	const [data, isLoading] = useFetch(
-		"https://www.json-generator.com/api/json/get/cfQXOXbwgO?indent=2",
-	);
-
-	return (
-		<div className="grid">
-			<div data-g="3">
-				<Filter />
-			</div>
-			<div data-g="9">
-				{isLoading ? (
-					<p>Loading...</p>
-				) : (
-					<table>
-						<thead>
-							<tr>
-								<th>Name</th>
-								<th>Email</th>
-							</tr>
-						</thead>
-						<tbody>
-							{data.map(({ _id, name, email }) => (
-								<tr key={_id}>
-									<td>
-										<Link to={`/users/${_id}`}>{name}</Link>
-									</td>
-									<td>{email}</td>
-								</tr>
-							))}
-						</tbody>
-					</table>
-				)}
-			</div>
-		</div>
-	);
+type User = {
+	id: number;
+	email_address: string;
+	user_id: string;
+	first_name: string;
+	last_name: string;
+	email_verified: boolean;
+	blocked: boolean;
 };
+
+interface StateType {
+	data: Array<User>;
+}
+
+interface PropsType {}
+
+export class UsersList extends Component<PropsType, StateType> {
+	constructor(props: PropsType) {
+		super(props);
+		this.state = {
+			data: [],
+		};
+	}
+
+	fetchData = async (url: string) => {
+		const response = await fetch(url);
```

Comment:
Is now a good time to think about how we're handling errors? For example are we going to to global/root level error handling or are we going to do it at a per component level?